### PR TITLE
Merging multiple branches

### PR DESCRIPTION
**Pull request by baileyjarvis2814**
# Multi-branch merge

## Description
This PR merges multiple branches
## Changes Made
 -Applied context login to mock authentication for login page.
 -Began styling login page component.
 -Created data folder with mock user data, ticketMaster/spotify data
 -Applied basic styling layout
## Testing
 -Testing not yet initialized
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [ ] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

### CODE REVIEW

**Review by KyleMBoomer — APPROVED** [review]
After completing the merge conflicts as a group, I'm happy to merge this into main. We still have quite a bit of work to do with regard to rendering cards and shaping up the styling.

**Comment by peterkimpk1** [comment]
- Webscraped script ld=application/json from specific ticketmaster artist page for ticketmaster data. Most artists did not have upcoming shows so the data was mocked only for artists with upcoming shows

**Review by peterkimpk1 — APPROVED** [review]

File: src/Components/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -1,10 +1,57 @@
...
+  const [isLoggingIn, setIsLoggingIn] = useState(false)
+  const [email, setEmail] = useState('')
+  const [error, setError] = useState('')
+  const navigate = useNavigate()
+
+  const handleLoginClick = () => {
+    setIsLoggingIn(true)
+  }
+
+  const handleFormSubmit = () => {
```
Comment (Sterling47) [inline src/Components/LoginPage/LoginPage.tsx:18]: Nice Job on setting up the handleSubmit Function

**Comment by Sterling47** [comment]
Things are starting finally come together.
Lets get some test written to get the full cycle integration working.

**Pull request merged**